app/domain/group/privacy_policy_finder.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Pfadibewegung Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Group::PrivacyPolicyFinder
+
+  def initialize(group, person)
+    @group = group
+    @person = person
+  end
+
+  def self.for(group: nil, person: nil)
+    new(group, person)
+  end
+
+  def acceptance_needed?
+    !already_accepted? && groups.any?
```

Comment:
Hmm. Aktuell ist einmal akzeptiert eigentlich "für immer mit allen Updates akzeptiert". Könnte man hier nicht recht leicht den Vergleich von "neueste DSE von einer Gruppe" und "Akzeptanzdatum auf der Person" machen? Oder ist das bewusst out-of-scope?